src/components/AppLayout/ContextualLinks/useContextualLink.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { renderHook } from "@testing-library/react-hooks";
+import React, { ReactNode } from "react";
+import { IntlProvider } from "react-intl";
+
+import { useContextualLink } from "./useContextualLink";
+
+jest.mock("@dashboard/components/ProductAnalytics/useAnalytics", () => ({
+  useAnalytics: jest.fn().mockReturnValue({ trackEvent: jest.fn() }),
+}));
+
+const wrapper = ({ children }: { children: ReactNode }) => (
+  <IntlProvider locale="EN">{children}</IntlProvider>
+);
+
+describe("useContextualLink", () => {
+  it("should return correct link for staff_members", () => {
+    // Act
+    const { result } = renderHook(() => useContextualLink("staff_members"), { wrapper });
+
+    // Assert
+    expect(result).toMatchSnapshot();
```

Comment:
Please don't use snapszots, there are several reasons why it is a not good idea, for instance:
- you don't see the assertion, just "match snapshot", so you don't know what you expect
- the generated markup is prone to many changes, often not even related to your test, eg changing class or link, most likely you accept it right away
- quite hard to maintain due to vast markup after some time you won't even remember what you want to "test/match" here 

and more,

There are plenty other ways of how you can test it. Clearly in your test you expect some link to be properly set, this seem to be your main assertion. Just mock the ContextualLineLink and assert if the props were passed properly, that's it